### PR TITLE
TASK: Removes lastVisitedNode javascript and handling

### PR DESCRIPTION
**Pull request by markusguenther**
In NeosCMS, we had implemented JavaScript to track the last visited page. However, this approach is not considered a best practice, and the feature was not functional with Neos 9.0 - we remove the last visited page tracking JavaScript to improve the codebase's maintainability and ensure compatibility with Neos 9.0.

If you override the fusion already, you can remove the override with Neos 9.0.

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

File: Neos.Neos/Classes/Controller/Module/Administration/SitesController.php [hunk 1]
```diff
@@ -644,7 +644,7 @@ public function deactivateDomainAction(Domain $domain)
      * @param string $format The format to use for the redirect URI
      * @return void
      */
-    protected function unsetLastVisitedNodeAndRedirect(
+    protected function redirectToAction(
```
Comment (mhsdesign) [inline Neos.Neos/Classes/Controller/Module/Administration/SitesController.php]: it seems we can instead always just call `$this->redirect` instead and remove this `redirectToAction` completely?

Comment (markusguenther) [reply Neos.Neos/Classes/Controller/Module/Administration/SitesController.php]: Had exactly the same thought and decided to be lazy 😂

Comment (mhsdesign) [reply Neos.Neos/Classes/Controller/Module/Administration/SitesController.php]: haha 3 seconds :P 0e7f1dfe85b303500569fbafa93061bd964bef92

**Review by ahaeslich — APPROVED** [review]

**Review by nezaniel — APPROVED** [review]
Looks good to me

**Comment by dlubitz** [comment]
Are there any plans or ideas to implement a new way to do this in Neos 9?

**Comment by kitsunet** [comment]
IMHO it could become a flowpack package @dlubitz

**Comment by dlubitz** [comment]
Yeah, could be. But if I check the changes for the removal, I get this feeling this will become an AOP hell 🙈

**Pull request merged**